src/window.ts

```diff
@@ -99,16 +139,15 @@ getCachedReferences(window);
  * that will be installed (via the membrane) as global descriptors in
  * the red realm.
  */
-function aggregateWindowDescriptors(
-    redDescriptors: PropertyDescriptorMap,
+export function aggregateWindowDescriptors(
```

Comment:
any particular reason for exporting this function now? I don't think it is used outside of this file!

Reply:
We export the other aggregate helper so figured it might have been overlooked.

Reply:
we don't use this... lets remove the export.

Reply:
Doh I removed the other aggregate helper 😃 
I'll get this one too.